blockchain/network.py

```diff
@@ -700,22 +700,22 @@ def get_unsent_transfer_ids(self, transfer_to=None, verification_id=None):
 
         return unsent_transfer_ids
 
-    def transfer_data(self, received, unreceived):
+    def transfer_data(self, transfer_to, received, unreceived):
         """ mark verifications as sent and return unsent verifications """
         verifications = []
         node_unreceived = []
         all_guids = received + unreceived
 
         # retrieve ids that calling node is claiming as unreceived
         for guid in unreceived:
```

Comment:
again, why are we querying the transfer table for ids we already have? I think we should get rid of the node_unreceived array altogether and just use the` unreceived` list when retrieving unreceived records below.